v2/openapi/fb-provider-off-exchange-api.yaml

```diff
@@ -0,0 +1,28 @@
+openapi: 3.0.0
+info:
+  title: 'Fireblocks Provider Off-Exchange API'
+  version: 0.0.1
+
+tags:
+  - name: offExchange
+    description: Endpoints used by external providers (exchanges) calling Fireblocks for off-exchange collateral flows.
+  - name: collateralWebhooks
+    description: Webhook notifications sent by Fireblocks to exchanges for collateral events.
```

Comment:
The webhooks implemented by providers should be specified in `fb-provider-collateral-api.yaml`. The endpoints could be under the usual `collateral` tag, or if you prefer `collateralNotifications`, or `collateralWebhooks`. It's only one endpoint and I wouldn't bother with a separate tag, personally.